src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/TelemetryUtility.cs

```diff
@@ -211,7 +212,7 @@ public static string ToJsonArrayOfTimingsInSeconds(IEnumerable<TimeSpan> sourceT
             sb.Append("[");
             foreach (var item in sourceTimings)
             {
-                sb.Append(item.TotalSeconds);
+                sb.Append(item.TotalSeconds.ToString(CultureInfo.InvariantCulture));
```

Comment:
If this is now always passing `CultureInfo.InvariantCulture`, do we really need a test to verify that?  The code is obviously correct and the test below will now depend on us setting a culture to make sure that whatever culture is currently configured won't actually be used.  But the code is already explicit, perhaps we don't need a test?

Reply:
If you think that the risk of this bug being reintroduced is smaller than the maintenance cost of those tests, why not?

Reply:
Yeah let's undo the test changes, we don't think any verification is needed for different cultures since the code is always passing `InvariantCulture`.

Reply:
I've reverted the test changes.